.github/workflows/ci-cd-pipeline.yml

```
@@ -26,4 +31,7 @@ jobs:
   # アーカイブビルド検証
   build-archive:
     name: Build Unsigned Archive
-    uses: ./.github/workflows/build-unsigned-archive.yml 
\ No newline at end of file
+    uses: ./.github/workflows/build-unsigned-archive.yml
+    secrets:
+      TEAM_ID: ${{ secrets.TEAM_ID }}
+      APPLE_ID: ${{ secrets.APPLE_ID }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)